package.json

```diff
@@ -1,15 +1,11 @@
 {
   "name": "@essex/pbi-base",
-  "version": "0.9.0",
+  "version": "1.0.0",
+  "private": true,
```

Comment:
Why are we setting private = True here?  Is this just temporary while everything is being re-worked to prevent publishing before its ready?